tower-cli/towercli/commands/provision.py

```diff
@@ -140,68 +140,68 @@ def add_args(argparser):
-            parser_error("private_key path invalid.")
+            parser_error("Private_key path invalid.")
 
     if args.keyboard_layout:
         if re.match(r'^[a-zA-Z]{2}$', args.keyboard_layout) is None:
-            parser_error(message="Keyboard layout invalid. Must be 2 chars.")
+            parser_error(message="Keyboard layout invalid. Must be 2 characters long.")
     
     if args.keyboard_variant:
         if re.match(r'^[a-zA-Z0-9-_]{2,32}$', args.keyboard_variant) is None:
-            parser_error(message="Keyboard layout invalid. Must be alphanumeric between 2 and 32 chars.")
+            parser_error(message="Keyboard layout invalid. Must be alphanumeric between 2 and 32 characters in length.")
 
     if args.timezone:
         if re.match(r'^[a-zA-Z-\ ]+\/[a-zA-Z-\ ]+$', args.timezone) is None:
-            parser_error(message="Timezone invalid. Must be in <Area>/<City> format. eg. Europe/Paris.")
+            parser_error(message="Timezone invalid. Must be in `<Area>/<City>` format, e.g. `Europe/Paris`.")
 
     if args.lang:
         if  re.match(r'^[a-z]{2}_[A-Z]{2}$', args.lang) is None:
-            parser_error(message="Lang invalid. Must be in <lang>_<country> format. eg. en_US.")
+            parser_error(message="Language invalid. Must be in `<lang>_<country>` format, e.g. `en_US`.")
 
     if args.image:
         if not os.path.exists(args.image):
-            parser_error(message="Invalid path for the image.")
+            parser_error(message="Invalid path to the image.")
         ext = args.image.split(".").pop()
         if os.path.exists(args.image) and ext not in ['img', 'xz']:
-            parser_error(message="Invalid extension for image path (only `xz`or `img`).")
+            parser_error(message="Invalid extension for image path. Must be either `xz`or `img`.")
 
     if args.ifname:
         interaces = utils.get_interfaces()
         if args.ifname not in interaces:
-            parser_error(message=f"Invalid network interface. Must be one of: {', '.join(interaces)}")
+            parser_error(message=f"Invalid network interface. Must be one of: {', '.join(interaces)}.")
     
     if args.name[0] == "thinclient":
         parser_error(message="You can't use `thinclient` as host name.")
     elif args.name[0] == sshconf.ROUTER_HOSTNAME:
         if not args.wlan_ssid:
-            parser_error(message="You must provide a wifi SSID for the router.")
+            parser_error(message="You must provide a WiFi SSID for the router.")
         if not args.wlan_password:
-            parser_error(message="You must provide a wifi password for the router.")
+            parser_error(message="You must provide a WiFi password for the router.")
     else:
         if args.online == args.offline:
-            parser_error(message="You must use one and only one of the argument `--online` and `--offline`.")
+            parser_error(message="You must specify `--online` or `--offline` and not both.")
```

Comment:
hum.. @adamkrellenstein this message is also displayed when the user does not use either flag (false == false) and not only when he uses both at the same time.

Reply:
fixed!